Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -766,3 +766,77 @@ void JitArm64::slwx(UGeckoInstruction inst)
 			ComputeRC(gpr.R(a), 0);
 	}
 }
+
+void JitArm64::rlwimix(UGeckoInstruction inst)
+{
+	INSTRUCTION_START
+	JITDISABLE(bJITIntegerOff);
+
+	int a = inst.RA, s = inst.RS;
+	u32 mask = Helper_Mask(inst.MB, inst.ME);
+
+	if (gpr.IsImm(a) && gpr.IsImm(s))
+	{
+		u32 res = (gpr.GetImm(a) & ~mask) | (_rotl(gpr.GetImm(s), inst.SH) & mask);
+		gpr.SetImmediate(a, res);
+		if (inst.Rc)
+			ComputeRC(res, 0);
+	}
+	else
+	{
+		if (mask == 0 || (a == s && inst.SH == 0))
+		{
+			// Do Nothing
+		}
+		else if (mask == 0xFFFFFFFF)
+		{
+			gpr.BindToRegister(a, a == s);
+			if (a != s)
+				MOV(gpr.R(a), gpr.R(s));
```

Comment:
This mov isn't required if inst.SH != 0, you can use gpr.R(s) directly as source.